conf/application.conf

```diff
@@ -108,6 +108,10 @@ metrics {
 
 # Microservice specific config
 
+feature-switch {
+  desIf_Migration.enabled = false
```

Comment:
Do we really want it to be disabled here? 
Long term it will be enabled (and can be enabled in test envs) so leaving this empty will allow the default (enabled) and we more easily the config tidy by just removing overrides in production/QA.